docs/guides/universal-receiver-delegate/accept-reject-assets.md

```diff
@@ -0,0 +1,213 @@
+---
+sidebar_label: 'Accept & Reject Assets'
+sidebar_position: 2
+---
+
+# Accept & Reject Assets
+
+Each user can create its own **custom Universal Receiver Delegate** contract that holds its own logic to be executed once the **[`universalReceiver(..)`](../../standards/smart-contracts/lsp0-erc725-account.md#universalreceiver)** function on his profile is called. 
+
+![LSP1UniversalReceiverDelegate-Guide](/img/guides/UniversalReceiverDelegate-Guide.jpeg)
+
+## Rejecting all Assets
+
+In order to **reject all the assets** that are being transferred to the profile, we need to create a Universal Receiver Delegate contract that reverts when it's the case of asset transfer (LSP7 & LSP8). The [`typeId`](../../standards//smart-contracts/lsp0-erc725-account.md#universalreceiver) is the parameter that will give us more context on the call being made.
+
+
+*e.g.*
+- If typeId is **[`0xdbe2c314e1aee2970c72666f2ebe8933a8575263ea71e5ff6a9178e95d47a26f` _TYPEID_LSP7_TOKENSRECIPIENT](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP7DigitalAsset/LSP7Constants.sol#L13)**, then we know that we are receiving an LSP7 Token.
+
+- If typeId is **[`0xc7a120a42b6057a0cbed111fbbfbd52fcd96748c04394f77fc2c3adbe0391e01` _TYPEID_LSP8_TOKENSRECIPIENT](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP8IdentifiableDigitalAsset/LSP8Constants.sol#L21)**, then we know that we are receiving an LSP8 Token.
+
+
+
+### Step1 - Deploy contract through Remix
+
+The first step is to navigate to **[Remix's website](https://remix.ethereum.org/)** and create a new solidity file under the **contracts** folder.
+
+![Creating Universal Receiver Delegate in Remix](/img/guides/remix-creating-file.jpeg)
+
+After creating the **UniversalReceiverDelegate.sol** file, copy the code snippet below inside the file created. This code snippet will be responsible for rejecting all LSP7 & LSP8 assets being transferred to your profile.
+
+```js title="UniversalReceiverDelegate.sol - Solidity Code snippet of the URD that reject all assets"
```

Comment:
Is `sol` working ?

It is js atm - We added `sol` as an additional param here:
https://github.com/lukso-network/docs/blob/main/docusaurus.config.js#L170

Maybe you can give it a try?

Reply:
Ah, nice! Thanks for the suggestion 👍